src/pages/validate.tsx

```diff
@@ -29,119 +29,195 @@ import validateLocalDocument from "../lib/validateLocalDocument";
 import { ValidationResult } from "../lib/types";
 
 function ClientIdentifierValidator() {
-  const [documentJson, setDocumentJson] = useState("");
-  const [clientIdentifierUri, setClientIdentifierUri] = useState("");
-  const [isFetchingAndValidating, setIsFetchingAndValidating] = useState(false);
+  // get the document from search parameter, if supplied
```

Comment:
```suggestion
  // Get the document from search parameter, if supplied
```
Please double-check that comments start with upper case